lib/engine/step/g_1822/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require_relative '../buy_sell_par_shares'
+require_relative 'bidbox_auction'
+
+module Engine
+  module Step
+    module G1822
+      class BuySellParShares < BuySellParShares
+        include BidboxAuction
+
+        attr_accessor :bidders, :bid_actions
+
+        def actions(entity)
+          actions = super
+          actions << 'bidbox_minors'
+          actions << 'bidbox_concessions'
+          actions << 'bidbox_privates'
+          actions << 'bidding_tokens'
```

Comment:
Yeah, the extra actions feel unnecessary. Maybe I'm missing something.

Reply:
Also, don't include if the player can't lay or increase a bid due to available cash. This is needed to auto-skip that player if they have no actions.

Reply:
As commented above, i can change just to have the bid left.

Reply:
Yes good ide. There are a few more places where autopass can be added.